### DISCUSSION TITLE
No 3/4 fingers vertical swipe nor pinch

### DISCUSSION THREAD

**Discussion by JacoLclodo** [bulletmark/libinput-gestures#363] [Q&A]
libinput-gestures: session xfce+x11 on Linux-6.9.0-1-MANJARO-x86_64-with-glibc2.39, python 3.12.3, libinput 1.25.0
Hash: 1e084483b086645ffe79e86f3b59961d
Gestures configured in ~/.config/libinput-gestures.conf:
swipe left 3 xdotool key Alt+Right
swipe right 3 xdotool key Alt+Left
swipe left 4 xdotool key Ctrl+Alt+Right
swipe right 4 xdotool key Ctrl+Alt+Left
swipe down 3 xdotool key Ctrl+Alt+D
libinput-gestures: device /dev/input/by-path/platform-i8042-serio-1-event-mouse(event14): SynPS/2 Synaptics TouchPad
libinput-gestures is installed.
libinput-gestures is set up as a desktop application.
libinput-gestures is currently running as a desktop application.
libinput-gestures is set to autostart as a desktop application.
libinput-gestures is using custom configuration file.

touchpad on lenovo T460 : everything works except : 3/4 fingers vertical swipe and pinch
3 fingers vertical swipe Xorg.0.log : "libinput_event_touch6] (EE) event14 - SynPS/2 Synaptics TouchPad: libinput bug: invalid gesture event GESTURE_EVENT_HOLD_TIMEOUT in state GESTURE_STATE_HOLD"

Any Idea ?

Where could I disable the 3 fingers tap "paste" command ?

**Comment by bulletmark** [comment]
Not really sure what you are asking but please perform the steps in the [TROUBLESHOOTING section](https://github.com/bulletmark/libinput-gestures#troubleshooting). In particular, note step 8 there.

**Reply by JacoLclodo** [reply]
The "swipe down 3 xdotool key Ctrl+Alt+D" does not work nor any vertical swipe or pinch while the keyboard combination keys does.
The "libinput debug-gui" and "libinput debug-events" show the gesture is recognised. 

Don't know if it's a configuration problem or an xfce4 problem.

` xinput --list-props "SynPS/2 Synaptics TouchPad"
Device 'SynPS/2 Synaptics TouchPad':
	Device Enabled (184):	1
	Coordinate Transformation Matrix (186):	1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000
	libinput Tapping Enabled (316):	1
	libinput Tapping Enabled Default (317):	0
	libinput Tapping Drag Enabled (318):	1
	libinput Tapping Drag Enabled Default (319):	1
	libinput Tapping Drag Lock Enabled (320):	0
	libinput Tapping Drag Lock Enabled Default (321):	0
	libinput Tapping Button Mapping Enabled (322):	1, 0
	libinput Tapping Button Mapping Default (323):	1, 0
	libinput Natural Scrolling Enabled (324):	1
	libinput Natural Scrolling Enabled Default (325):	0
	libinput Disable While Typing Enabled (326):	1
	libinput Disable While Typing Enabled Default (327):	1
	libinput Scroll Methods Available (328):	1, 1, 0
	libinput Scroll Method Enabled (329):	1, 0, 0
	libinput Scroll Method Enabled Default (330):	1, 0, 0
	libinput Click Methods Available (331):	1, 1
	libinput Click Method Enabled (332):	1, 0
	libinput Click Method Enabled Default (333):	1, 0
	libinput Middle Emulation Enabled (334):	0
	libinput Middle Emulation Enabled Default (335):	0
	libinput Accel Speed (336):	0.460000
	libinput Accel Speed Default (337):	0.000000
	libinput Accel Profiles Available (338):	1, 1, 1
	libinput Accel Profile Enabled (339):	1, 0, 0
	libinput Accel Profile Enabled Default (340):	1, 0, 0
	libinput Accel Custom Fallback Points (341):	<no items>
	libinput Accel Custom Fallback Step (342):	0.000000
	libinput Accel Custom Motion Points (343):	<no items>
	libinput Accel Custom Motion Step (344):	0.000000
	libinput Accel Custom Scroll Points (345):	<no items>
	libinput Accel Custom Scroll Step (346):	0.000000
	libinput Left Handed Enabled (347):	0
	libinput Left Handed Enabled Default (348):	0
	libinput Send Events Modes Available (301):	1, 1
	libinput Send Events Mode Enabled (302):	0, 0
	libinput Send Events Mode Enabled Default (303):	0, 0
	Device Node (304):	"/dev/input/event7"
	Device Product ID (305):	2, 7
	libinput Drag Lock Buttons (349):	<no items>
	libinput Horizontal Scroll Enabled (350):	1
	libinput Scrolling Pixel Distance (351):	15
	libinput Scrolling Pixel Distance Default (352):	15
	libinput High Resolution Wheel Scroll Enabled (353):	1
`

**Comment by bulletmark** [comment]
Did you read my comment and try step 8? Does your `xdotool` command typed on the command line work as you want?
